python/dglke/tests/test_dataset.py

```diff
@@ -0,0 +1,243 @@
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+    def test_udd_ofr_rids(self):
+        extra_train = ['0\t2\t1\n']
+        files = gen_udd_files(extra_train=extra_train)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+        extra_valid = ['0\t2\t1\n']
+        files = gen_udd_files(extra_valid=extra_valid)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+        extra_test = ['0\t2\t1\n']
+        files = gen_udd_files(extra_test=extra_test)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+    def test_udd_error_nids(self):
+        extra_train = ['-1\t0\t1\n']
+        files = gen_udd_files(extra_train=extra_train)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+        extra_train = ['0\t0\t-1\n']
+        files = gen_udd_files(extra_train=extra_train)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+        extra_valid = ['-1\t0\t1\n']
+        files = gen_udd_files(extra_valid=extra_valid)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+        extra_test = ['-1\t0\t1\n']
+        files = gen_udd_files(extra_test=extra_test)
+        with self.assertRaises(AssertionError):
+            dataset = get_dataset('./',
+                                  'udd_test',
+                                  'udd_hrt',
+                                  '\t',
+                                  files)
+        cleanup(files)
+
+    def test_udd_error_nids(self):
```

Comment:
it has the same name as the method above?

Reply:
It should be test_udd_error_rids.